docs/getting-started.md

```diff
@@ -192,7 +192,7 @@ LOGFILE="./firecracker.log"
 touch $LOGFILE
 
 # Set log file
-curl -X PUT --unix-socket "${API_SOCKET}" \
+sudo curl -X PUT --unix-socket "${API_SOCKET}" \
```

Comment:
Alternatively, could we change the permissions of the API_SOCKET? i.e. `chmod a+rw $API_SOCKET`?

Reply:
it could work, `$API_SOCKET` requires root because we started firecracker with root.

For consistency, I think we can either use sudo everywhere (so it's easy to setup etc), or we don't use sudo at all, which requires user to setup the access to `/dev/kvm`. 

what do you think?

Reply:
I think the `sudo` starting Firecracker is OK, as it does need some special permissions. But the ones for curl are not strictly required, as we could just allow writing to the socket.

Another option is to just run everything after `sudo su`, but it's equivalent to running all with `sudo`.